<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Состояние КРН Нарышкино</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="Statewgt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Refresh_but">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить состояние</string>
    </property>
   </widget>
   <widget class="QPushButton" name="onBut">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Включить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="offBut">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Отключить</string>
    </property>
   </widget>
   <widget class="QLabel" name="ip_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Настройки</string>
     </property>
     <addaction name="Address"/>
     <addaction name="Port"/>
    </widget>
    <addaction name="menu_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Address">
   <property name="text">
    <string>IP-адрес</string>
   </property>
  </action>
  <action name="Port">
   <property name="text">
    <string>Порт</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
